python/pyspark/sql/tests/plot/test_frame_plot_plotly.py

```diff
@@ -64,20 +73,15 @@ def test_bar_plot(self):
         self._check_fig_data("bar", fig["data"][0], ["A", "B", "C"], [10, 30, 20], "int_val")
         self._check_fig_data("bar", fig["data"][1], ["A", "B", "C"], [1.5, 2.5, 3.5], "float_val")
 
-    def test_barh_plot(self):
```

Comment:
why removing this test?

Reply:
My bad.. removed by mistake while rebasing